java/code/src/com/redhat/rhn/frontend/action/kickstart/EditPackagesAction.java

```diff
@@ -151,7 +151,7 @@ private void transferEdits(KickstartData ksdata, DynaActionForm form,
 
                 // if noBase is checked and the current package is @base, ignore it
                 PackageName pn = PackageFactory.lookupOrCreatePackageByName(pkg);
-                if ((pn.getName().toLowerCase().replaceAll("\\s", "")
+                if ((pn.getName().toLowerCase().replace("\\s", "")
```

Comment:
`\\s` is a regexp group. You cannot replace this `replaceAll` using a `replace` here since this is more than just characters.